# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1248854938 -7200
#      Wed Jul 29 10:08:58 2009 +0200
# Node ID 49f890499ebccc4ec29293890427e9bc4b17f6fa
# Parent  2ebd0717c12dd91c2b28d7b9263929c2eb5252ea
fix typo in gnuplot_binary

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2009-07-29  Jaroslav Hajek  <highegg@gmail.com>
+
+	* plot/gnuplot_binary.m: Fix typo.
+
 2009-07-27  Jaroslav Hajek  <highegg@gmail.com>
 
 	* optimization/fzero.m: Use "global" parameter in str2func.
 	* optimization/fsolve.m: Ditto.
 	* optimization/fminunc.m: Ditto.
 
 2009-07-24  John W. Eaton  <jwe@octave.org>
 
diff --git a/scripts/plot/gnuplot_binary.in b/scripts/plot/gnuplot_binary.in
--- a/scripts/plot/gnuplot_binary.in
+++ b/scripts/plot/gnuplot_binary.in
@@ -13,17 +13,17 @@
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Loadable Function} {[@var{prog}, @var{args}] =} gnuplot_binary ()
-## @deftypefnx {Loadable Function} {[@var{old_prog}, @var{old_args}] =} gnuplot_binary (@var{new_prog}, @varg{arg1}, @dots{})
+## @deftypefnx {Loadable Function} {[@var{old_prog}, @var{old_args}] =} gnuplot_binary (@var{new_prog}, @var{arg1}, @dots{})
 ## Query or set the name of the program invoked by the plot command
 ## and any arguments to pass to it.  Previous arguments are returned
 ## as a cell array.
 ## The default value @code{\"gnuplot\"}.  @xref{Installation}.
 ## @end deftypefn
 
 ## Author: jwe
 
